client/src/views/util.js

```diff
@@ -8,13 +8,29 @@ const DEFAULT_PRECISION = 0
 
 const pad = n => n < 10 ? '0'+n : n
 
-export const formatTime = unix => {
-  const time = new Date(unix*1000)
-      , tzOffset = time.getTimezoneOffset() * -1
-
-  return `${time.getFullYear()}-${pad(time.getMonth() + 1)}-${pad(time.getDate())}`
-       + ` ${pad(time.getHours())}:${pad(time.getMinutes())}:${pad(time.getSeconds())}`
-       + ` GMT${tzOffset == 0 ? '' : (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
+export const formatTime = (unix, dateType) => {
+  // Return Time only
+  if(dateType === "time"){
+    const time = new Date(unix*1000)
+
+    return `${time.getFullYear()}-${pad(time.getMonth() + 1)}-${pad(time.getDate())}`
+        + ` ${pad(time.getHours())}:${pad(time.getMinutes())}:${pad(time.getSeconds())}`
+  
+  // Return Timezone only
+  }else if(unix === undefined && dateType === "timezone"){
+        const time = new Date()
+        , tzOffset = time.getTimezoneOffset() * -1
+        return `GMT${tzOffset == 0 ? '' : (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
+  
+  // Default Return Time and Timezone.... dataType === undefined. 
+  }else{
+      const time = new Date(unix*1000)
+        , tzOffset = time.getTimezoneOffset() * -1
+
+    return `${time.getFullYear()}-${pad(time.getMonth() + 1)}-${pad(time.getDate())}`
+        + ` ${pad(time.getHours())}:${pad(time.getMinutes())}:${pad(time.getSeconds())}`
+        + ` GMT${tzOffset == 0 ? '' : (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
+  }
```

Comment:
I would go with something like this to reduce duplication, and to use a boolean flag instead of a dataType enum.

```js
export const formatTimezone = unix => {
  const tzOffset = new Date(unix*1000).getTimezoneOffset() * -1;
  return `GMT${tzOffset == 0 ? '' : (tzOffset < 0 ? '' : '+') + (tzOffset/60)}`
}

export const formatTime = (unix, with_tz) => {
  const time = new Date(unix*1000)

  return `${time.getFullYear()}-${pad(time.getMonth() + 1)}-${pad(time.getDate())}`
       + ` ${pad(time.getHours())}:${pad(time.getMinutes())}:${pad(time.getSeconds())}`
       + (with_tz ? ' ' +formatTimezone(unix) : '')
}
```

Reply:
Changes made. Thank you.